front-end/src/components/main-page/TranscriptUploadModal.jsx

```diff
@@ -12,9 +13,21 @@ class TranscriptUploadModal extends React.Component {
     this.state = {
       file: null,
       isFileValid: null,
+      showOverrideModal: false,
+      override: false,
     };
   }
 
+  toggleOverrideModal = () => {
+    this.setState({showOverrideModal: !this.state.showOverrideModal});
+  }
+
+  handleUploadOverride = () => {
+    this.setState({override: true},() => {
```

Comment:
```suggestion
    this.setState({override: true}, () => {
```
Little nitpick with the spacing lol